### PR TITLE
Add Publish from Azure DevOps

### PR DESCRIPTION
**Pull request by kale-swapnil**

### CODE REVIEW

**Comment by kale-swapnil** [comment]
@dotnet-policy-service agree

**Comment by vanillajonathan** [comment]
Looks good!

There could also be instructions for deploying to Azure Static Web Apps instead of GitHub Pages.

**Review by yufeih — APPROVED** [review]
Thank you @kale-swapnil !

File: docs/docs/publish-azure-devops.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# Setting up DocFX with Azure DevOps to Deploy to GitHub Pages
```
Comment (yufeih) [inline docs/docs/publish-azure-devops.md:1]: Do we also want this to appear in the [Table of Content](https://github.com/dotnet/docfx/blob/main/docs/docs/toc.yml)

Comment (kale-swapnil) [reply docs/docs/publish-azure-devops.md:1]: I think so. Let me know if that needs to be done, and I will update it.

Comment (yufeih) [reply docs/docs/publish-azure-devops.md:1]: Yes please

**Comment by vanillajonathan** [comment]
You can also publish to Azure Static Web Apps.

https://learn.microsoft.com/en-us/azure/static-web-apps/get-started-portal?tabs=vanilla-javascript&pivots=azure-devops

**Comment by filzrev** [comment]
> You can also publish to Azure Static Web Apps.

Azure Static Web Apps has [`Total File Count` quota](https://learn.microsoft.com/en-us/azure/static-web-apps/quotas) to `15,000`.
- `https://github.com/Azure/static-web-apps/issues/520`

So there is concern about reaching in this quota when deploying large document sites.  
(Especially when using `"memberLayout": "separatePages"` options)

**Pull request merged**